scripts/test_examples.sh

```diff
@@ -8,15 +8,16 @@ PREV_IFS="$IFS"
 # arguments
```

Comment:
please do not commit the scripts/test_examples.sh.7z binary

Reply:
Done